fastlane/lanes/localization.rb

```diff
@@ -388,19 +388,38 @@
   #####################################################################################
   lane :download_translations do
     # WordPress strings
+    wordpress_res_dir = File.join('WordPress', 'src', 'main', 'res')
     android_download_translations(
-      res_dir: File.join('WordPress', 'src', 'main', 'res'),
+      res_dir: wordpress_res_dir,
       glotpress_url: APP_SPECIFIC_VALUES[:wordpress][:glotpress_appstrings_project],
-      locales: WP_APP_LOCALES,
-      lint_task: 'lintWordpressVanillaRelease'
+      locales: WP_APP_LOCALES
     )
+
     # Jetpack strings
+    jetpack_res_dir = File.join('WordPress', 'src', 'jetpack', 'res')
     android_download_translations(
-      res_dir: File.join('WordPress', 'src', 'jetpack', 'res'),
+      res_dir: jetpack_res_dir,
       glotpress_url: APP_SPECIFIC_VALUES[:jetpack][:glotpress_appstrings_project],
-      locales: JP_APP_LOCALES,
-      lint_task: 'lintJetpackVanillaRelease'
```

Comment:
See the whole rationale for removing the `lint_task` here in the commit message for 77d16992a46e4d484bf5f54e3ad8705f2ce70918

TL;DR: lint would have been too soon, and we plan to make that be done by CI very soon (i.e. once we migrate Release steps from CircleCI to Buildkite and can restore the dependency between lint and release CI steps) instead of having to wait forever for it to run it locally when we do betas.